frontend/src/patientApp/timeOfTest/TestResult.tsx

```diff
@@ -0,0 +1,134 @@
+import { connect, useSelector } from "react-redux";
+
+import { formatFullName } from "../../app/utils/user";
+import { RootState } from "../../app/store";
+import { Patient } from "../../app/patients/ManagePatients";
+
+const serializeResult = (result: string) => {
```

Comment:
There is actually a `TestResult` type exported from the `QueueItem` component which could be used here